lib/simplifyExpression/collectAndCombineSearch/LikeTermCollector.js

```diff
@@ -220,6 +223,19 @@ function getTermsForCollectingMultiplication(node) {
 }
 
 // A helper function for getTermsForCollectingMultiplication
+// e.g. nthRoot(x, 2), append '2': nthRootNode to terms dictionary
```

Comment:
I'd name this something more descriptive than just '2'. Like 'nthRoot2' or something, not sure.